Makefile

```diff
@@ -80,17 +80,17 @@ GO_PACKAGE_PATH ?= github.com/${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}
 GOLANGCI_LINT_BIN=./out/golangci-lint
 .PHONY: lint
 ## Runs linters on Go code files and YAML files
-lint: lint-go-code lint-yaml
+lint: ./vendor lint-go-code lint-yaml
```

Comment:
Vendor directory is already committed to git. So this is not required.

Reply:
Ok, I'll remove it.